pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -137,30 +152,37 @@ async def run(self):
                     advisories["extras"] = self.create_advisory("RHBA", "image", "extras")
                 if advisories.get("metadata", 0) <= 0:
                     advisories["metadata"] = self.create_advisory("RHBA", "image", "metadata")
-            _LOGGER.info("Created advisories: %s", advisories)
 
-            _LOGGER.info("Saving the advisories to ocp-build-data...")
-            advisories_changed = await self.save_advisories(advisories)
-
-            if advisories_changed:
-                _LOGGER.info("Sending an Errata live ID request email...")
-                self.send_live_id_request_mail(advisories)
+        advisories_changed = advisories != group_config.get("advisories", {})
+
+        _LOGGER.info("Ensuring JIRA ticket for release %s...", self.release_name)
+        jira_issue_key = group_config.get("release_jira")
+        jira_template_vars = {
+            "release_name": self.release_name,
+            "x": self.release_version[0],
+            "y": self.release_version[1],
+            "z": self.release_version[2],
+            "release_date": self.release_date,
+            "advisories": advisories,
+            "candiate_nightlies": self.candidate_nightlies,
```

Comment:
typo

Reply:
Seems the typo is also in the JIRA template so we didn't notice that... I will hold off on the typo fix at this moment until we fix the releases for the next week so we can continue to use the job in my hackspace.

Reply:
Solved.